Modules/signalmodule.c

```diff
@@ -143,6 +135,16 @@ static HANDLE sigint_event = NULL;
 static PyObject *ItimerError;
 #endif
 
+Py_LOCAL_INLINE(PyObject *)
+GetHandler(int i) {
```

Comment:
Usually in the C code, we use names like get_handler().

Reply:
Hmm, right. I'm not used to the CPython style anymore :-)